src/Microsoft.AspNet.Razor/Generator/LiteralAttributeCodeGenerator.cs

```diff
@@ -62,9 +63,9 @@ public override bool Equals(object obj)
         {
             var other = obj as LiteralAttributeCodeGenerator;
             return other != null &&
```

Comment:
as discussed offline, consistency between `Equals()` and `GetHashCode()` is the goal. that's not the same as doing the same (somewhat inefficient in these base classes) things in all `Equals()` implementations.